src/one_dragon/base/operation/one_dragon_app.py

```
@@ -148,3 +148,8 @@ def after_switch_account(self) -> OperationRoundResult:
 
     def after_operation_done(self, result: OperationResult):
         Application.after_operation_done(self, result)
+        # 一条龙运行结束，递增运行计数
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)